src/components/checkbox/checkboxGroup/checkboxGroup.tsx

```diff
@@ -0,0 +1,14 @@
+import classNames from 'classnames';
+import type { ComponentProps } from 'react';
+
+export interface ICheckboxGroupProps extends ComponentProps<'div'> {}
+
+export const CheckboxGroup: React.FC<ICheckboxGroupProps> = (props) => {
```

Comment:
This is almost a useless component!

Reply:
Lol, that's fine, it sets the spacing between the elements 😁